### PR TITLE
Frontier pallets storage process

### PR DESCRIPTION
**Pull request by AsceticBear**
Prefer https://github.com/darwinia-network/darwinia-2.0/issues/14 in a separate pr after the balances state process is finished because this `Approves` goes to the pallet-asset storage in the end.

### CODE REVIEW

**Review by hackfisher — DISMISSED** [review]

**Comment by AurevoirXavier** [comment]
I'll make some impovement later.

File: tool/state-processor/src/balances/mod.rs [hunk 1]
```diff
@@ -30,7 +30,7 @@ impl Processor {
 		// GWEI)); solo_kton_locks.iter_mut().for_each(|(_, v)| v.iter_mut().for_each(|l| l.amount
 		// *= GWEI));
 
-		log::info!("take para balances total issuances and locks");
+		log::info!("take para `Balances::TotalIssuance` and `Balances::Locks`");
```
Comment (AsceticBear) [inline tool/state-processor/src/balances/mod.rs:33]:
```rs
// solo_ring_locks.iter_mut().for_each(|(_, v)| v.iter_mut().for_each(|l| l.amount *=
		// GWEI)); solo_kton_locks.iter_mut().for_each(|(_, v)| v.iter_mut().for_each(|l| l.amount
		// *= GWEI));
```
Can we delete this?

Comment (AurevoirXavier) [reply tool/state-processor/src/balances/mod.rs:33]: Maybe leave it there for some tips.

**Review by AsceticBear — COMMENTED** [review]
LGTM

**Pull request merged**